prog/postgres/postgres_timeline_nexus.rb

```diff
@@ -64,13 +64,19 @@ def before_run
   end
 
   label def wait
+    leader = postgres_timeline.leader
+    if leader.nil? && postgres_timeline.backups.empty? && Time.now - postgres_timeline.created_at > 10 * 24 * 60 * 60
```

Comment:
I would save `postgres_timeline.backups` to a local variable and use it here and in the code below, as it requires an API call.